content/references/http-websocket-apis/public-api-methods/ledger-methods/ledger.md

```diff
@@ -191,27 +107,28 @@ The response follows the [standard format][], with a successful result containin
 
 | `Field`                        | Type    | Description                       |
 |:-------------------------------|:--------|:----------------------------------|
-| `ledger`                       | Object  | The complete header data of this ledger. |
-| `ledger.account_hash`          | String  | Hash of all account state information in this ledger, as hex |
-| `ledger.accountState`          | Array   | (Omitted unless requested) All the [account-state information](ledger-data-formats.html) in this ledger. |
```

Comment:
I was of the understanding that we mark a field as `[deprecated]` and gracefully remove it from the responses. Have we informed the community sufficiently in advance? 
My apologies if this was communicated before, I'm not aware of this development.

Reply:
This _is_ the communication. The code will still work without changes, but this updates the docs to communicate the deprecation.

Reply:
oh okay :+1